AgentDiscoveries-Frontend/app/src/components/admin/location-form.jsx

```diff
@@ -92,14 +126,16 @@ export default class LocationForm extends React.Component {
 
         const body = {
             siteName: this.state.siteName,
+            longitude: this.state.longitude,
+            latitude: this.state.latitude,
             location: this.state.location,
             timeZone: this.state.timeZone,
             regionId: this.state.regionId ? this.state.regionId : null
         };
 
         const request = this.props.id
-            ? apiPut('locations', body, this.props.id)
-            : apiPost('locations', body);
+            ? apiPut('locations', body, this.props.id) //do this if location ID is present
```

Comment:
I would much rather split this into an if statement than have these comments here - I'm happy that they're useful for development but I'd very against keeping them for production